routes/auth/user/getUsers.ts

```diff
@@ -2,15 +2,15 @@ import { db, event } from "#src/utils";
 
 export const getUsers = async () => {
   const limit = event.queryStringParameters?.limit ?? 10;
-  const offset = event.queryStringParameters?.page ?? 0;
+  const offset = event.queryStringParameters?.offset ?? 0;
```

Comment:
Using query params here so it's easy to share these links around and see the same view.